### DISCUSSION TITLE
IndexError: tuple index out of range

### DISCUSSION THREAD

**Discussion by wlindqwister** [PMEAL/porespy#694] [Q&A]
On a fresh install of PoreSpy (v2.1), I get the following error when importing the package. I've tried to update docrep, but that does not seem to fix the error. Could it be something related to my python/Anaconda installation?

---------------------------------------------------------------------------
IndexError                                Traceback (most recent call last)
<ipython-input-2-d90270dc439c> in <module>
----> 1 import porespy

~\anaconda3\lib\site-packages\porespy\__init__.py in <module>
     65 from . import filters
     66 from . import metrics
---> 67 from . import networks
     68 from . import generators
     69 from . import simulations

~\anaconda3\lib\site-packages\porespy\networks\__init__.py in <module>
     25 """
     26 
---> 27 from ._funcs import add_boundary_regions
     28 from ._funcs import generate_voxel_image
     29 from ._funcs import label_phases

~\anaconda3\lib\site-packages\porespy\networks\_funcs.py in <module>
      1 import numpy as np
----> 2 import openpnm as op
      3 import scipy.ndimage as spim
      4 from skimage.segmentation import find_boundaries
      5 from skimage.morphology import ball, cube, disk, square

~\anaconda3\lib\site-packages\openpnm\__init__.py in <module>
     62 from . import phases
     63 from . import physics
---> 64 from . import algorithms
     65 from . import materials
     66 from . import topotools

~\anaconda3\lib\site-packages\openpnm\algorithms\__init__.py in <module>
     11 
     12 from .GenericAlgorithm import GenericAlgorithm
---> 13 from .GenericTransport import GenericTransport
     14 
     15 from .ReactiveTransport import ReactiveTransport

~\anaconda3\lib\site-packages\openpnm\algorithms\GenericTransport.py in <module>
    100 @docstr.get_sectionsf('GenericTransport', sections=['Parameters'])
    101 @docstr.dedent
--> 102 class GenericTransport(GenericAlgorithm):
    103     r"""
    104     This class implements steady-state linear transport calculations

~\anaconda3\lib\site-packages\openpnm\algorithms\GenericTransport.py in GenericTransport()
    215         self.settings.update(**kwargs)
    216 
--> 217     @docstr.get_full_descriptionf(base='GenericTransport.reset')
    218     @docstr.get_sectionsf(base='GenericTransport.reset', sections=['Parameters'])
    219     @docstr.dedent

~\anaconda3\lib\site-packages\docrep\decorators.py in deprecated(self, *args, **kwargs)
    125                     return getattr(self, replacement)(*args, **kwargs)
    126                 else:
--> 127                     return func(self, *args, **kwargs)
    128 
    129         else:

~\anaconda3\lib\site-packages\docrep\__init__.py in get_full_descriptionf(self, *args, **kwargs)
    989                 removed_in="0.4.0")
    990     def get_full_descriptionf(self, *args, **kwargs):
--> 991         return self.get_full_description(base=args[0], *args[1:], **kwargs)
    992 
    993     @deprecated('get_extended_summary', "0.3.0", replace=False,

IndexError: tuple index out of range

**Comment by jgostick** [comment]
Hard to say what this issue is, have never seen this one myself.  I"m guessing you have incompatible versions of the various packages installed maybe?

**Reply by ma-sadeghi** [reply]
It could be the docrep package. Try upgrading to the latest version.
